src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const thItems = document.querySelectorAll('th');
+const trItems = tbody.querySelectorAll('tr');
+
+function toNum(str) {
+  return +str.replace(/[$,]/g, '');
+};
+
+[...thItems].map((el, i) => {
+  el.addEventListener('click', () => {
```

Comment:
use event delegation on table headers instead of applying event listener on each item